src/main/java/training/day2/HomeworkExercise1.java

```diff
@@ -0,0 +1,12 @@
+package training.day2;
+
+public class HomeworkExercise1 {
+    public static void main(String[] args){
+        String[] words = {"alpha", "avatar", "trouble", "birch", "salon"};
+
+        for(String str : words){
```

Comment:
Space between ) and {. And even after for. This is for better formatting of the code.